staging/src/k8s.io/legacy-cloud-providers/aws/instances.go

```diff
@@ -91,12 +91,17 @@ func (name KubernetesInstanceID) MapToAWSInstanceID() (InstanceID, error) {
 }
 
 // mapToAWSInstanceID extracts the InstanceIDs from the Nodes, returning an error if a Node cannot be mapped
-func mapToAWSInstanceIDs(nodes []*v1.Node) ([]InstanceID, error) {
+// and skipping Nodes that do not match with the specified target node labels
+func mapToAWSInstanceIDs(nodes []*v1.Node, targetNodeLabels map[string]string) ([]InstanceID, error) {
```

Comment:
is this been called? if not, let's not modify this. since mapToAWSInstanceIDsTolerant is supposed to allow ignore nodes.

And TBH, i'm kind of favor not change `mapToAWSInstanceIDsTolerant`, and do the label filter within in findInstancesForELB directly prior to call mapToAWSInstanceIDsTolerant. Given the function name didn't imply filter nodes.